packages/gafl-webapp-service/src/pages/summary/licence-summary/__tests__/route.spec.js

```diff
@@ -50,288 +39,250 @@ jest.mock('../../../../processors/mapping-constants.js', () => ({
-        }
-      }),
-      i18n: {
-        getCatalog: () => ({
-          licence_type_radio_salmon: 'Salmon and sea trout'
-        })
-      },
-      url: {
-        search: ''
-      },
-      path: '',
-      locale: 'en'
+const getMockBlueBadgePermission = () => ({
+  ...getMockPermission(),
+  concessions: [
+    {
+      type: mappingConstants.CONCESSION.DISABLED,
+      proof: {
+        type: mappingConstants.CONCESSION_PROOF.blueBadge,
+        referenceNumber: 'AB1 CDE 0 1234F5678'
+      }
     }
+  ]
+})
 
-    it('should return a summary table with required data for page', async () => {
-      mockTransactionCacheGet.mockImplementationOnce(() => ({
-        birthDateStr: '1st January 1946',
-        concessionProofs: {
-          NI: 'National Insurance Number',
-          blueBadge: 'Blue Badge',
-          none: 'No Proof'
-        },
-        cost: 6,
-        disabled: true,
-        hasExpired: false,
-        hasJunior: false,
-        isContinuing: false,
-        isRenewal: true,
-        licenceLength: '12M',
-        licenceStartDate: '2021-07-01',
-        licenceType: 'Salmon and sea trout',
-        licenceTypeStr: 'Salmon and sea trout',
-        licensee: {
-          birthDate: '1946-01-01',
-          firstName: 'Graham',
-          lastName: 'Willis'
-        },
-        numberOfRods: '3',
-        permit: {
-          cost: 6
-        },
-        startAfterPaymentMinute: 30,
-        startTimeString: '0.00am (first minute of the day) on 1 July 2021',
-        uri: {
-          clear: '/buy/new',
-          dateOfBirth: '/buy/date-of-birth',
-          disabilityConcession: '/buy/disability-concession',
-          licenceLength: '/buy/licence-length',
-          licenceStartDate: '/buy/start-kind',
-          licenceToStart: '/buy/start-kind',
-          licenceType: '/buy/licence-type',
-          name: '/buy/name'
-        }
-      }))
-      const result = await getData(mockRequest)
-      expect(result).toMatchSnapshot()
-    })
+const getMockContinuingPermission = () => ({
+  ...getMockPermission(),
+  licenceToStart: 'another-date',
+  renewedEndDate: '2022-11-10'
+})
 
-    it.each([
-      [NAME.uri],
-      [LICENCE_LENGTH.uri],
-      [LICENCE_TYPE.uri],
-      [LICENCE_TO_START.uri],
-      [DATE_OF_BIRTH.uri],
-      [DISABILITY_CONCESSION.uri],
-      [RENEWAL_START_DATE.uri],
-      [LICENCE_TO_START.uri],
-      [NEW_TRANSACTION.uri]
-    ])('addLanguageCodeToUri is called with the expected arguments', async uri => {
-      const permission = {
-        permit: {
-          cost: 1
-        },
-        licensee: {
-          birthDate: '1996-01-01'
-        },
-        isRenewal: true
-      }
+describe('licence-summary > route', () => {
+  beforeEach(jest.clearAllMocks)
 
-      mockTransactionCacheGet.mockImplementationOnce(() => permission)
+  describe('sets from summary on status cache', () => {
+    it('adds LICENCE_SUMMARY_SEEN to status.fromSummary for a renewal', async () => {
```

Comment:
in this describe could we call statusCacheSet = jest.fn() once and use for all tests within?

Reply:
No - as far as possible, you shouldn't share mocks between tests